cime_config/config_component.xml

```diff
@@ -1405,6 +1405,22 @@
     <desc>rof2ocn runoff mapping file</desc>
   </entry>
 
+  <entry id="OCN2WAV_SMAPNAME">
```

Comment:
I think you want something of the form
  <entry id="ocn2wav_map">
    <type>char</type>
    <category>mapping</category>
    <group>MED_attributes</group>
    <desc>ocn to wav mapping, 'unset' or 'idmap' are normal possible values</desc>
    <values>
      <value>unset</value>
      <value samegrid_ocn_wav='true'>idmap</value>
    </values>
  </entry>

And set samegrid_ocn_wav in buildnml. It doesn't make sense to have idmap be the default value if the two are on different grids.

Reply:
However, do we really want to have these as xml variables - or just add them in as namelist variables in case the user might want to change them?

Reply:
I think maybe there is a format error in your comment - if you meant to include xml can you wrap it in <CODE></CODE>.
Also I don't see a variable samegrid_ocn_wav in the cmeps buildnml.   I'm not sure what you want.